.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -1,38 +1,22 @@
 name: Wheel::Linux::ARM
 on:
+  pull_request: # for tests
```

Comment:
Should we remove this?

Reply:
will try it later and get back to you guys. @mlxd @AmintorDusko

Reply:
Pip install of whl files worked for me @mlxd and @AmintorDusko on m1